pkg/cri/resource-manager/requests.go

```diff
@@ -152,6 +152,7 @@ func (m *resmgr) RunPod(ctx context.Context, method string, request interface{},
 			m.policy.ReleaseResources(c)
 			c.UpdateState(cache.ContainerStateStale)
 		}
+		m.cache.DeletePod(p.GetID())
```

Comment:
nit: this line should be part of the first patch(?)

Reply:
Definitely.